src/SeleniumLibrary/keywords/browsermanagement.py

```diff
@@ -469,6 +483,13 @@ def _make_chrome(self, remote, desired_capabilities, profile_dir):
             webdriver.Chrome, webdriver.DesiredCapabilities.CHROME, remote,
             desired_capabilities)
 
+    def _make_headless_chrome(self, remote, desired_capabilities, profile_dir):
+        options = webdriver.ChromeOptions()
+        options.set_headless()
+        return self._generic_make_driver(
+            webdriver.Chrome, webdriver.DesiredCapabilities.CHROME, remote,
+            desired_capabilities, options=options)
```

Comment:
Calling `_make_chrome` here like `_make_ff` is called by `_make_headless_ff` above would probably be a good idea especially for consistency reasons.